### PR TITLE
Testing Your Ruby Code - Tic Tac Toe

### PR DESCRIPTION
**Pull request by julian3493**
###We create test cases for all the public methods in the class GAME

- Test to get_move() method created
- Test to winner? method created
- Test to draw? method created

### CODE REVIEW

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳
Hi @julian3493 and @mke2111, you did very well on this project.  Most requirements have done 💯, Kindly fix the following issues.

- [x] Kindly don't disable the **rubocop** rules on the _game.rb_ file

[Optional]

- [ ] Use `require_relative`  for referring to files within your directory on the _spec/game_spec.rb_ file
- [x] Kindly remove the _.gitkeep_ file from the lib directory

Once you fix the issues, kindly submit another code review 
 Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/)

File: spec/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# spec/game_spec.rb
+
+require '../lib/game.rb'
```
Comment (layzrd) [inline spec/game_spec.rb:3]: `require_relative '../lib/game.rb'`

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳
Hi @julian3493 and @mke2111, you did very well on this project. Most requirements have done 👌, Kindly fix the following issues.

- [x] Some of the unit tests are failing, thoughtfully fix the issues

[Optional]

- [ ] Kindly practice require_relative for referring files within your directory on the spec/game_spec.rb file

`require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.
 Happy coding 👨‍💻
 [Abdulaziz Redi Ali](https://azizali.ml/)

**Review by Srjnnnn — APPROVED** [review]
### ### :tada: :tada: Status: Approved :tada: :tada: :rotating_light: 
Hey @410AngelaVu and @julian3493. There are no more issues. You can merge this pull request and move on.

- [ ] Optional
You can use require_relative instead of requiring in your code when you just refer to an internal file.

Good job :+1: 

Eyüp Sercan UYGUR

**Pull request merged**